src/parser/validator.rs

```diff
@@ -381,8 +381,8 @@ impl<'cmd> Validator<'cmd> {
         conflicts: &mut Conflicts,
     ) -> bool {
         debug!("Validator::is_missing_required_ok: {}", a.get_id());
-        let conflicts = conflicts.gather_conflicts(self.cmd, matcher, a.get_id());
-        !conflicts.is_empty()
+        let overrides = conflicts.gather_overrides(self.cmd, matcher, a.get_id());
```

Comment:
merge conflict with 85ecb3e895d17ff391be6dc43a8e9944cb318cef and 4a34b9dd43ffc349cedd4be6d8928b625a06f096